docs/Basics-Layout.md

```diff
@@ -1,17 +1,17 @@
 ---
 id: basics-layout
-title: Layout
+title: Layout with Flexbox
 layout: docs
 category: The Basics
-permalink: docs/basics-layout.html
+permalink: docs/flexbox.html
```

Comment:
I think we don't need to worry about breaking the bookmarks to that page.
Who would bookmark it?